src/main/resources/static/index.html

```diff
@@ -1,13 +1,28 @@
-<!DOCTYPE HTML>
+<html xmlns:th="http://localhost" xmlns:layout="http://www.w3.org/1999/xhtml">
 <html>
   <head>
-    <title>Demo Shop</title>
+    <title>8-BLISS</title>
```

Comment:
Does this have to be this name? Can't we rename it to something like `Diffblue e-commerce Demo Shop` or something?

Reply:
I like your idea of storing the name in variable that can be easily modified. Ill keep a note of that for future developments. 

As for the designs and the name, at this current time I don't think we will be changing it.